server/src/main/java/org/candlepin/model/Product.java

```diff
@@ -185,7 +185,7 @@
     @ElementCollection
     @BatchSize(size = 32)
     @CollectionTable(name = "cp2_product_content", joinColumns = @JoinColumn(name = "product_uuid"))
-    @Column(name = "element")
+    @Column(name = "content_uuid")
     @LazyCollection(LazyCollectionOption.EXTRA) // allows .size() without loading all data
```

Comment:
Out of curiosity, what was the purpose of changing this? It seems out of band from this PR. Just looking for some context.